tests/0109/parser-error.txt

```diff
@@ -0,0 +1 @@
+ExpectedToken(["an identifier"], Some("c"), (7, 32)) -> Parse error: unexpected token `c`, expecting an identifier on line 7 column 32
```

Comment:
this error is weird, it should be "expecting a variable"

Reply:
oh... type parser is trying to parse this! this is a bug!

Reply:
see: https://3v4l.org/IRQnS, `php-parser-rs` thinks `&` means intersection types!

Reply:
fixed! :)